docs/faq.rst

```diff
@@ -14,7 +14,7 @@ etcd is a consistent distributed key-value store. Mainly used as a separate coor
 How to pronounce etcd?
 ======================
 
-etcd is pronounced ``/etˈ-sē-dē/`` or ``/etˈ-see-dee/``, and means distributed ``etc`` directory.
+etcd is pronounced ``/ˈɛtsiːdiː/``, and means distributed ``etc`` directory.
```

Comment:
Just my 2 cents - if we really decide to do it then may be add another `or` to original text instead of removing it. or perhaps be we just don't want this change. @gyuho @xiang90 can tell more.